pkg/portal/libstripe/model.go

```diff
@@ -12,6 +12,7 @@ import (
 const (
 	MetadataKeyAppID          = "app_id"
 	MetadataKeyPlanName       = "plan_name"
+	MetadataKeyVersion        = "version"
```

Comment:
Because I do not want to parse the plan name. And it is also not a must to include the version in plan name. `version` is for grouping a group of plans into a group, so that some products which applies to all plan (Such as sms usage) can be easily applied to the whole "group".